CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
 - Remove deprecated `config.ServiceTelemetry` (#5565)
 - Remove deprecated `config.ServiceTelemetryLogs` (#5565)
 - Remove deprecated `config.ServiceTelemetryMetrics` (#5565)
+- Removed `HasFlag`, `MetricDataPointFlagsNone`, and `MetricDataPointFlagNoRecordedValue`. Use new Flags API to check for Flags and to set and remove Flags (#5602)
```

Comment:
These are deprecated not deleted, correct?

Also, please move them to unreleased section.

Reply:
Good catch, need to update this again.

Reply:
Done.